src/main/java/com/wanderersoftherift/wotr/abilities/attachment/AbilityCooldowns.java

```diff
@@ -82,22 +82,30 @@ public LongRange getCooldown(AbilitySource slot) {
     public Stream<CooldownInfo> getCooldowns() {
         return cooldowns.entrySet()
                 .stream()
-                .filter(x -> x.getValue().to() > getGameTime())
+                .filter(x -> x.getValue().to() > getGameTime() * 1000)
                 .map(x -> new CooldownInfo(x.getKey(), x.getValue()));
     }
 
     /**
      * @param source
      * @return Whether the given slot is on cooldown
      */
-    public boolean isOnCooldown(AbilitySource source) {
-        return cooldowns.getOrDefault(source, LongRange.EMPTY).to() > getGameTime();
+    public boolean isOnCooldown(AbilitySource source, int margin) {
```

Comment:
I would suggest having a `boolean isCooldown(AbilitySource source)` overload that calls this with the default margin.